### PR TITLE
#153728037 Add multiple alert methods

### PR DESCRIPTION
**Pull request by Thegaijin**
#### What does this PR do?
Setting up healthchecks to be able to send out important information concerning the cron jobs through multiple channels

#### Description of Task to be completed?
Integration of several alert methods in the system such that the system is able to send a user alerts of all those selected platforms when a cron job is pinged and when it goes down.

#### How should this be manually tested?
Open health checks.io on your localhost
Navigate to the integrations page. 
* For email, click on the add integrations button in the email row, add the email you would like the notifications to be sent to. An email is sent to that email inbox to verify the setup. Click on the button in the email to confirm set up.

* For sms, click on the add integrations button in the sms row. You will be redirected to a page where you can add the phone number you would like to get your notifications on. Add the number and click on the save integration button. The number should be registered with twilio for it to work.

* For slack, click on the add integrations button in the sms row,  you are redirected to a page where you can select the channel to which the notifications will be sent to from your list of available channels 

#### The files changed/created include:
Created: 
add_sms.html - Added UI for the page where the phone number is added
.env - Added the twilio configuration settings

Edited
hc/settings.py - Added the smtp server settings and the email details
hc/local_settings.py - Added the SLACK configurations
hc/api/transports.py - Added the SMS model
hc/front/urls.py - Added the URL that redirects to the sms configuration page
hc/front/views.py - Added the function that loads the page where the number to sms is added
hc/templates/front/channels.html - Added sms to the list of configurable channel options
hc/front/tests/test_add_channel.py 

#### What are the relevant pivotal tracker stories?
[#153728037](https://www.pivotaltracker.com/story/show/153728037)

Screenshots
the channels
channels

cron jobs in various states
Cron jobs in different states

screen shot 2018-01-18 at 23 38 34
Email notifications

slack notifications
Slack notifications

sms notifications
SMS notifications

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Twilio API credentials
+export TWILIO_ACCOUNT_SID=AC807f0beda34c3b254e0cf785a42420af
+export TWILIO_AUTH_TOKEN=4de3c67a78763ae5c4e67a2cadce45f9
+
+# Twilio sending number
+export TWILIO_NUMBER=+13143000470
+
```
Comment (mwaz) [inline .env:7]: This too is private information could be stored directly on Heroku as variables defined
Tokens should always remain private, reason why we place them in the environment 
all of the variables can be stored on Heroku variables where the Database URL variable is defined

You should send a sample.env not the .env file :-)

**Comment by mwaz** [comment]
This will not run if you do not add a worker process 
`./manage.py sendalerts` on heroku Procfile 

otherwise LGTM

**Comment by anyric** [comment]
👍 
LGTM

**Review by anyric — APPROVED** [review]
👍 LGTM

**Review by jimmykimani — APPROVED** [review]

File: .env (hunk 1 above)
Comment (Thegaijin) [reply .env:7]: Noted. Thanks for the heads up

**Pull request merged**